quic requires unix-time >= 0.4.12

For `instance Generic UnixTime`, otherwise the following compile error will raise:

```
       /tmp/stack-48044dc96d65b90a/quic-0.1.16/Network/QUIC/Packet/Token.hs:32:10: error: [GHC-39999]                                                                                                                                                                             
           • No instance for ‘Generic UnixTime’                                                                                                                                                                                                                                   
               arising from a use of ‘Codec.Serialise.Class.$dmencode’                                                                                                                                                                                                            
           • In the expression: Codec.Serialise.Class.$dmencode @(UnixTime)                                                                                                                                                                                                       
             In an equation for ‘encode’:                                                                                                                                                                                                                                         
                 encode = Codec.Serialise.Class.$dmencode @(UnixTime)                                                                                                                                                                                                             
             In the instance declaration for ‘Serialise UnixTime’                                                                                                                                                                                                                 
          |                                                                                                                                                                                                                                                                       
       32 | instance Serialise UnixTime                                                                                                                                                                                                                                           
          |          ^^^^^^^^^^^^^^^^^^                                                                                                                                                                                                                                           
```

--- a/quic.cabal
+++ b/quic.cabal
@@ -139,7 +139,7 @@ library
         stm >= 2.5 && < 2.6,
         data-default-class >= 0.1.2 && < 0.2,
         fast-logger >= 3.2.2 && < 3.3,
-        unix-time >= 0.4.11 && < 0.5,
+        unix-time >= 0.4.12 && < 0.5,
         iproute >= 1.7.12 && < 1.8,
         network >= 3.1.4 && < 3.2,
         network-byte-order >= 0.1.7 && < 0.2,

